docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+services:
+  shutter-service-api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    depends_on:
+      db:
+        condition: service_healthy
+    environment:
+      - DB_HOST=db
+      - DB_PORT=5432
+      - DB_USER=postgres
+      - DB_PASSWORD=${DB_PASSWORD}
+      - DB_NAME=keyper_db
+      - DB_SSL_MODE=disable
+      - RPC_URL=${RPC_URL}
+      - SHUTTER_REGISTRY_CONTRACT_ADDRESS=${SHUTTER_REGISTRY_CONTRACT_ADDRESS}
+      - KEY_BROADCAST_CONTRACT_ADDRESS=${KEY_BROADCAST_CONTRACT_ADDRESS}
+      - KEYPER_SET_MANAGER_CONTRACT_ADDRESS=${KEYPER_SET_MANAGER_CONTRACT_ADDRESS}
+      - KEYPER_HTTP_URL=${KEYPER_HTTP_URL}
+      - SIGNING_KEY=${SIGNING_KEY}
+      - SERVER_PORT=23003
+    ports:
+      - "23003:23003"
+    command: ["./shutter-service-api"]
+
+  db:
```

Comment:
why do we need database configured here? we would be using db setup by keyper

Reply:
added for testing purposes